### DISCUSSION TITLE
I have a question about "Github Pages"

### DISCUSSION THREAD

**Discussion by shidijun** [community/community#144739] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

(My English is poor.Pardon please.)
My "Github Page" is displaying the "RENDME.md".Can I let it display what I want html file?
﻿
<a href="https://shidijun.github.io/blog">The website</a>
﻿
我的“Github页面”显示“RENDME.md”。我可以让它显示我想要的html文件吗？
﻿
<a href="https://shidijun.github.io/blog">所述网站</a>
﻿

**Comment by mohanamisra** [comment, marked as answer]
Try renaming your `homepage.html` file to `index.html`.

GitHub pages is looking for an "entry point" into your project in order to host it, which is typically an html file named `index.html`. In its absence (which is the case for you), it hosts the contents of the `README.md`.

Check out [where the docs explain this in more detail](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#:~:text=Create%20the%20entry%20file%20for%20your%20site.%20GitHub%20Pages%20will%20look%20for%20an%20index.html%2C%20index.md%2C%20or%20README.md%20file%20as%20the%20entry%20file%20for%20your%20site.).

**Reply by shidijun** [reply]
I'll try it

**Reply by shidijun** [reply]
It's very successful!Thank you very much.

**Reply by mohanamisra** [reply]
No problem, glad to help.

PS: Good job on the blog !
